.travis.yml

```diff
@@ -2,7 +2,17 @@ language: generic
 
 script: bash ./deploy.sh
 
+branches:
+  only:
+    - gh-pages
```

Comment:
> I think we want master in the only list also, or general PRs might not cause Travis to run deploy.sh to check that the files compile.

Oh, I might be confused, but I thought we only wanted it on gh-pages to allow us to keep editing on master before deploying via gh-pages. I'll change it to master tho.